setup.py

```diff
@@ -29,13 +29,11 @@
     long_description=open("README.md").read() if exists("README.md") else "",
     install_requires=[
         "aiohttp",
-        "azure-core>=1.5.0",
+        "azure-core>=1.7.0",
         "azure-datalake-store>=0.0.46,<0.1",
         "azure-identity",
-        "azure-storage-blob",
+        "azure-storage-blob>=12.5.0",
```

Comment:
This is required by the use of BlobClient.exists, and the import from `_list_blobs_helper`.